### DISCUSSION TITLE
No packages for Linux

### DISCUSSION THREAD

**Discussion by ShadowOfPripyat** [FluxpointDev/MultiRPC#27] [Old]
Hello, i downloaded the tar.gz, and its just full of .DLL(s) and nothing to compile the program.
Could you provide us a .deb package?

Thank You!

**Comment by Azyyyyyy** [comment]
We plan to make packages (At least `.deb` and `PKGBUILD`) and installers when the rewrite is closer to being finished! For now the tar.gz is the only way to use this application.

**Comment by Azyyyyyy** [comment]
We now create .deb packages :D

**Reply by ShadowOfPripyat** [reply]
After a lot of months, i've just seen it. Nice job and keep goin ;)

**Comment by AntiGamerPlayzz** [comment]
imagine being on linux kid

**Comment by xXBuilderBXx** [comment]
Redoing MultiRPC
